packages/luna/src/components/TutorCard/TutorCardV1.tsx

```diff
@@ -0,0 +1,227 @@
+          <Typography
+            element="caption"
+            weight="bold"
+            className="tw-ellipsis tw-line-clamp-2 tw-text-natural-800 tw-underline"
+          >
+            {intl("tutors.card.label.read-more")}
+          </Typography>
+        </Link>
+
+        {!isNewTutor ? (
+          <div className={cn("tw-flex tw-gap-8 tw-my-4")}>
+            {studentCount ? (
+              <div className="tw-flex tw-flex-col tw-gap-1">
+                <Typography
+                  element="caption"
+                  weight="regular"
+                  className="tw-text-natural-800"
+                >
+                  {intl("tutors.card.label.students")}
+                </Typography>
+                <Typography
+                  element="body"
+                  weight="semibold"
+                  className="tw-text-natural-950"
+                >
+                  {formatNumber(studentCount)}
+                </Typography>
+              </div>
+            ) : null}
+
+            {lessonCount ? (
+              <div className="tw-flex tw-flex-col tw-gap-1">
+                <Typography
+                  element="caption"
+                  weight="regular"
+                  className="tw-text-natural-800"
+                >
+                  {intl("tutors.card.label.lessons")}
+                </Typography>
+                <Typography
+                  element="body"
+                  weight="semibold"
+                  className="tw-text-natural-950"
+                >
+                  {formatNumber(lessonCount)}
+                </Typography>
+              </div>
+            ) : null}
+
+            {rating ? (
+              <div className="tw-flex tw-flex-col tw-gap-1">
+                <Typography
+                  element="caption"
+                  weight="regular"
+                  className="tw-text-natural-800"
+                >
+                  {intl("tutors.card.label.rating")}
+                </Typography>
+                <div className="tw-flex tw-flex-row tw-items-center tw-gap-[5px] tw-h-full">
+                  <Typography
+                    element="tiny-text"
+                    weight="semibold"
+                    className="tw-inline-block tw-text-natural-950"
+                  >
+                    {rating}
+                  </Typography>
+                  <SStar
+                    width={15}
+                    height={15}
+                    className="[&>*]:tw-fill-warning-500"
+                  />
+                </div>
+              </div>
+            ) : null}
+          </div>
+        ) : null}
+
+        {!isEmpty(topics) ? (
+          <div className="tw-flex tw-mt-4 tw-gap-2 tw-flex-wrap tw-justify-start">
+            {topics.map((topic, idx) => {
+              if ((isNewTutor && idx < 6) || (!isNewTutor && idx < 3))
+                return (
+                  <Tooltip key={idx} content={<div>{topic}</div>}>
```

Comment:
It is not professional to render the topic like that without any styles. Please wrapp it in a typography and use the body variant.